heudiconv/main.py

```diff
@@ -108,6 +108,12 @@ def process_extra_commands(outdir, command, files, dicom_dir_template,
         ensure_heuristic_arg(heuristic)
         from .utils import get_heuristic_description
         print(get_heuristic_description(heuristic, full=True))
+    elif command == 'populate-intended-for':
+        for subj in subjs:
```

Comment:
I guess we could either read the `participants.tsv` or do a `glob` for `sub-*` in the `outdir`.
Which is preferred?

Reply:
since we would need to glob sessions anyways, ideally
- glob sub-*
- read from participants.tsv and if differs -- issue a warning
- glob session per each sub

Reply:
Addressed in 51d1a96d45fd599ab48eb1ed060ec65688146c37